compute/vm_disk.go

```diff
@@ -9,7 +9,8 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2017-03-30/compute"
+	disks "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-04-01/compute"
```

Comment:
Nice that multi-API allows us to do this :). It's not great that we have to totally rename but maybe this is the clearest way.

@marstr thoughts?

Reply:
The latest tag in the config file for compute doesn't actually include the latest compute APIs which support user-assigned identities, so we have to use disks from the previous release.

Reply:
This may be an example a scenario that would be improved if we published packages at a resource type level.